packages/dashboard/src/util/transformAlarms.ts

```diff
@@ -0,0 +1,29 @@
+import { AlarmQuery } from '@iot-app-kit/source-iotsitewise';
+import {
+  AlarmCompositeModelRequest,
+  AlarmData,
+} from '@iot-app-kit/react-components';
+
+const convertAlarmQueryToAlarmRequest = (alarmQuery: AlarmQuery) => {
+  return (
+    alarmQuery.alarmComponents?.reduce((results, alarmComponent) => {
+      results.push({
+        assetId: alarmQuery.assetId,
+        assetCompositeModelId: alarmComponent.assetCompositeModelId,
+      });
+      return results;
+    }, [] as AlarmCompositeModelRequest[]) ?? []
+  );
+};
+
+export const mapAlarmQueriesToRequests = (alarmQueries: AlarmQuery[]) =>
+  alarmQueries.flatMap((query) => convertAlarmQueryToAlarmRequest(query));
+
+export const transformAlarmForPropertyPanel = (alarm: AlarmData) => ({
+  propertyId: alarm.compositeModelId,
+  name: alarm.compositeModelName,
+  unit: alarm.inputProperty?.at(0)?.property.unit,
+  dataType: alarm.inputProperty?.at(0)?.property.dataType,
+  alias: undefined,
+  assetName: alarm.inputProperty?.at(0)?.property.name,
```

Comment:
Isn't this the name of the property not the asset?

Reply:
bigger issue with this component is that it doesnt work with our definition of alarms so im working around its definition

currently an asset property renders its name as `property_name (asset_name)` where asset name is taken from line 28 above, so im using this same format to render alarms as `alarm_name (alarm_property_name)`!

Reply:
Yeah that makes sense! Let's make that clearer either with comments or variable names, since that's not obvious here.